### ISSUE TITLE
Changes default border

### ISSUE THREAD

**Issue by julmot** [wenzhixin/bootstrap-table#2613]
I've just set up bootstrap-table. My table HTML:

``` html
<table class="table table-striped table-bordered">
  <thead>
    <!-- ... -->
  </thead>
  <tbody>
    <!-- ... -->
  </tbody>
</table>
```

JavaScript:

``` javascript
var $tables = $("main table"),
    $navbar = $("header nav");
$tables.each(function(){
    // Set individual ID necessary by BS table sticky-header
    var id = Math.random().toString(36).substring(7),
        topSpace = $navbar.outerHeight(true),
        $this = $(this);
    $this.attr("id", id).bootstrapTable({
        stickyHeader: true,
        stickyHeaderOffsetY: topSpace + "px",
        classes: $this.attr("class")
    });
});
```

First of
1. Making the ID necessary just for the stickyHeader component seems to be undocumented or at least not enough. When not initializing bootstrap-table with tables containing an "id" attribute, classes starting with "undefined" will be added to DOM elements
2. Adding the classes again by overwriting the defaults with `$this.attr("class")` seems to be ugly

Anyway, this isn't why I've created the issue. This will cause the following styles:
![after](https://cloud.githubusercontent.com/assets/8310677/18167045/c75a5d48-704e-11e6-8a3b-f7e4969d8a03.png)

While the table without bootstrap-table looks like:
![before](https://cloud.githubusercontent.com/assets/8310677/18167058/d0740bcc-704e-11e6-85cd-873115bd966e.png)

In summary, it will:
1. round the edges
2. add a double border to the entire table
3. replaces the 2px border-bottom of thead cells with 1px

This appears because of the containers wrapped around the table. While I could overwrite this with CSS myself, I expect that this doesn't happen.

**Comment by julmot** [comment]
Ping @wenzhixin

**Comment by Falseee** [comment]
I cant reproduce it, can you provide us an jsfiddle (if the issue still exists) ?

**Comment by Falseee** [comment]
I close this issue because we got no response/example within 7 days.
